src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -414,7 +414,6 @@ public void Track(ITelemetry telemetry)
                 if (string.IsNullOrEmpty(telemetry.Context.InstrumentationKey))
                 {
                     TelemetryDebugWriter.WriteTelemetry(telemetry);
```

Comment:
Can you please double check that debug output will still have this telemetry item written